app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager.rb

```diff
@@ -56,7 +56,7 @@ def self.description
   def self.provider_region_options
     ManageIQ::Providers::IbmCloud::VPC::Regions
       .all
-      .sort_by { |r| r[:description].downcase }
+      .sort_by { |r| r[:description]&.downcase }
```

Comment:
@Real-Omar-Afifi what does the UI show if there isn't a description?  I wonder if this is a possibility if we should be dropping any regions without a description in `ManageIQ::Providers::IbmCloud::VPC::Regions.all` instead?